app/components/HeaderContent.tsx

```diff
@@ -58,6 +58,9 @@ const HeaderContent = ({ session, settings }: { session?: Session; settings?: Ap
           <DeFiButton onPress={() => navigate(Routes.Cfp)} style={styles.button} compact>
             {t("cfp.link")}
           </DeFiButton>
+          <DeFiButton onPress={() => navigate(Routes.SpecialVoting)} style={styles.button} compact>
+            {t("specialVoting.link")}
+          </DeFiButton>
```

Comment:
Just curious - will this button get removed again or at least hidden?  Or do we want to always show special votings even if there are no special votings active?

Reply:
We'll remove it manually after a certain while. Currently we'll continue to show it to be transparent. 
We might use it thogh for future internal votings.